package/router/lex/tokens.go

```diff
@@ -41,9 +42,10 @@ func (tokens Tokens) At(i int) string {
 	for _, token := range tokens {
 		switch token.Type {
 		case PathToken, SlashToken:
-			for j := 0; j < len(token.Value); j++ {
+			chars := []rune(token.Value)
+			for j := 0; j < len(chars); j++ {
```

Comment:
Did you try ranging over `token.Value`? IIRC, you can do:

```go
for char := range token.Value {
  // do something
}
```

And it'll range over the runes.

Reply:
https://stackoverflow.com/a/18130921/145435

Reply:
Nice, I'll switch to that - I thought about it but didn't recognize that it "did the right thing" with the index key, i.e. skipping numbers for multi-byte runes.